src/domains/shared-app/api/session-wallet/v1/_base_policy.xml.tpl

```diff
@@ -1,225 +1,164 @@
 <policies>
     <inbound>
       <base />
-      <set-variable name="walletToken"  value="@(context.Request.Headers.GetValueOrDefault("Authorization", "").Replace("Bearer ",""))"  />
-      <!-- Perform PM start api call: for family&friends handling NPG flow must be activated only for 
-        users that have been enabled in pay-wallet-family-friends-user-ids named value (list of user id = PDV fiscal code tokenization)
-        For family&friends flow we expect that all clients will go with PM flow except the ones in the above list, that will be restricted
-        to few selected users. Anticipating PM start session will prevent further api calls.
-       -->
-      <!-- Session PM START-->
-      <send-request ignore-error="true" timeout="10" response-variable-name="pm-session-body" mode="new">
-          <set-url>@($"{{pm-host}}/pp-restapi-CD/v1/users/actions/start-session?token={(string)context.Variables["walletToken"]}")</set-url>
-          <set-method>GET</set-method>
+      <set-variable name="walletToken"  value="@(context.Request.Headers.GetValueOrDefault("Authorization", "").Replace("Bearer ",""))"  />     
+      <!-- user fiscal code tokenization with PDV START -->
+
+      <!-- Post Token PDV : START-->
+      <send-request ignore-error="true" timeout="10" response-variable-name="pdv-token" mode="new">
+        <set-url>${pdv_api_base_path}/tokens</set-url>
+        <set-method>PUT</set-method>
+        <set-header name="x-api-key" exists-action="override">
+            <value>{{wallet-session-personal-data-vault-api-key}}</value>
+        </set-header>
+        <set-body>@{
+          return new JObject(
+                  new JProperty("pii", (string)context.Variables["userFiscalCode"])
+              ).ToString();
+            }</set-body>
```

Comment:
this PDV call should be set after IO get user api call? here we don't have userFiscalCode yet (before userFiscalCode was taken from PM start user api call response body)

Reply:
you're right ... moved in commit 34346a0b9b10608857d6c16e88e79b90611246c3